airbyte-webapp/src/components/StreamTestingPanel/PageDisplay.module.scss

```diff
@@ -35,3 +35,16 @@
   max-height: 100%;
   overflow-y: auto;
 }
+
+// add a fade at the bottom of the tabPanel
+.tabPanel::after {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  bottom: 0;
+  left: 0;
+  pointer-events: none;
+  background-image: linear-gradient(to bottom, colors.$transparentColor, colors.$white 100%);
+  width: 100%;
+  height: 2vh;
```

Comment:
Is there a specific reason we're having this height in dependance to the viewport height, instead of just a static (e.g. `rem` or `px`) height?

Reply:
I was thinking that if the window height is small, the blur should be smaller so that more of the content is visible. But looking back on that decision, I think it makes more sense to just always keep the blur a small pixel value, since it doesn't really ever need to be large